webapp/graphite/render/datalib.py

```diff
@@ -43,12 +43,12 @@ def __init__(self, name, start, end, step, values, consolidate='average', tags=N
     else:
       self.tags = {'name': name}
       # parse for tags if a tagdb is configured and name doesn't look like a function-wrapped name
-      if STORE.tagdb and not re.match('^[a-z]+[(].+[)]$', name, re.IGNORECASE):
-        try:
+      try:
+        if STORE.tagdb and not re.match('^[a-z]+[(].+[)]$', name, re.IGNORECASE):
```

Comment:
was this throwing an exception?

Reply:
name was a `[]`, which obviously doesn't work well with re.match(). You can reproduce that with `time(time)` (instead of `time("time")`)

Reply:
Aha, fair enough